.github/cloudbuild/pos-pr-webhook-event.yaml

```diff
@@ -12,10 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-steps:
 ###########################################################
```

Comment:
Good point. At the moment this is the only cleanup task and I cannot think of anything else. It might or might not grow.

In the meantime, I came up with these files names based on the following thought train:
- These files are related to specific cloud build triggers; it is those triggers that uses them
- I want to make sure anyone who has access to the [cloud build triggers](https://pantheon.corp.google.com/cloud-build/triggers?project=point-of-sale-ci) for this repo can connect to these files easily
- So each of these file names have the same names [as the triggers](https://pantheon.corp.google.com/cloud-build/triggers?project=point-of-sale-ci)
- Also the trigger names are defined such that the it contains the repo name plus the kind of event when it's triggered.
- Since this trigger is a webhook event I names it `pos-pr-webhook-event`

Let me know if we should rethink this naming more or if this is good to go for now!

Reply:
this works for me, thanks for the clarification 👍